src/scss/_menulog.scss

```diff
@@ -28,6 +28,9 @@
     // =================================
     // These are the variables that should be used directly in our SCSS (globally)
 
+    //greys
+    $grey--dark                   : #333;
```

Comment:
This should be the same as `$grey--darkest` (which Menulog should have access to)?